aws-storage-s3/src/androidTest/java/com/amplifyframework/storage/s3/AWSS3StorageUploadTest.java

```diff
@@ -54,6 +54,7 @@
 /**
  * Instrumentation test for operational work on upload.
  */
+@Ignore("Contains test which either hang themselves, or hang the suite overall.")
```

Comment:
Same question.

Reply:
Yes; 2 commits, 1 PR, needed the tests working.